nixos/modules/services/network-filesystems/ipfs.nix

```diff
@@ -142,6 +142,35 @@ in
         description = "If set to true, the repo won't be initialized with help files";
```

Comment:
It sounds like the real solution here is just to set the "server" profile which applies these local IPs IIUC. I'm not completely opposed to making server the default but I don't think we should be applying this blacklist by default just because some hosting providers get grumpy.